README.build/Ubuntu-18.04.readme

```diff
@@ -0,0 +1,47 @@
+
+------------------------------------------
+Ubuntu Server 18.04 Cloud9 bootstrap build
+------------------------------------------
+
+   Instructions to build and run burr-tools from a Cloud9 amazonlinux environment.  This will build the burrTxt2 and burrGui applications, but I have only tested burrTxt2 for functionality, as there is no GUI available.
+
+Contact: Bryan Turner (bryan.turner@pobox.com or "brturn" on GitHub)
```

Comment:
I would rather point to the github issue tracker (same for other lines like this). People are more likely to post there and others can then already find solutions to their problems.